korgan/claim_consistency_guard.py

```
@@ -0,0 +1,136 @@
+            "Нельзя молча терять заявленный способ защиты: включите исполнимое требование по VERIFIED-норме и расчету либо явно оставьте документ preliminary с указанием, каких данных не хватает."
+        )
+
+    if costs_requested and not costs_in_prayer:
+        errors.append(
+            "Пользователь прямо просил взыскать судебные расходы, но соответствующего требования нет в разделе «ПРОШУ СУД». "
+            "Добавьте процессуально корректное требование о судебных расходах либо явно объясните, почему оно не может быть заявлено по текущим материалам."
+        )
+
+    if penalty_in_prayer and not _AMOUNT_RE.search(requests):
+        errors.append(
+            "В разделе «ПРОШУ СУД» заявлена денежная неустойка/пеня без конкретного размера или проверяемого расчета. "
```

Ruff (0.16.1)

[warning] String contains ambiguous `В` (CYRILLIC CAPITAL LETTER VE). Did you mean `B` (LATIN CAPITAL LETTER B)?

(RUF001)